topo/dva/faucet_block-yellow.yaml

```diff
@@ -692,8 +692,8 @@ dps:
       max_per_interval: 5
       send_interval: 5
 include:
-- uniform-acls.yaml
-- role-based-acls.yaml
+- uniform-acls_augmented.yaml
```

Comment:
what is this file, and what is this change for?  I can't tell just from this context and it's not obvious from the code how this file is used. I just want to make sure the abstraction isn't bleeding out inappropriatley.

Reply:
So Faucetizer reads ACLs from included ACL files (uniform-acls.yaml and role-based-acls.yaml), adds cookies to the ACLs and then writes the ACLs to other files (uniform-acls_augmented.yaml and role-based-acls_augmented.yaml), so that it does not overwrite the existing .yaml files, similar to faucet_structural.yaml and faucet.yaml.

Reply:
What is the purpose of the "faucet_block-yellow.yaml" file, how is that used?  Why is this change reflected in source-control... seems like the mechanism you describe should be internal only.

Reply:
why is this change in source control, though? The augmentation you describe should be internal and not exposed. Why is this change in faucet_block-yellow.yaml?  When is this file (faucet_block-yellow.yaml) used?

Reply:
That file is used for test_topology. Basically the testing script will compare generated file with that file and tests if they are identical.